lib/publiccloud/gce.pm

```diff
@@ -170,14 +170,17 @@ sub cleanup {
 
     my $region = $self->{provider_client}->{region};
     my $project = $self->{provider_client}->{project_id};
-    my $instance_id = $self->get_terraform_output(".vm_name.value[0]");
+    my $query = ".vm_name.value[0]";
+    my $instance_id = $self->get_terraform_output($query);
```

Comment:
What's the purpose of splitting this in 2 lines and creating a new variable, if the variable is never used again?

Reply:
This because that query was a point of attention in previous discussions, the right input-query could change depending on the needs, also Michele (for SAP) suggested avoid creating dependencies on data-structure. But for current publiccloud needs it is ok, so I let its definition separated from the call, open for future updates and customization needs.